package.json

```diff
@@ -6,6 +6,8 @@
   "version": "10.1.0-pre",
   "repository": "github:grafana/grafana",
   "scripts": {
+    "ludo-lint": "eslint --rule '@typescript-eslint/no-explicit-any: error' --rule '@grafana/no-aria-label-selectors: error' public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx",
```

Comment:
did you mean to commit these?

Reply:
no , it is for debuging ci issue, will remove when fixed